### PR TITLE
[Refactor] Use MMCV MODEL_REGISTRY

### PR DESCRIPTION
**Pull request by Wuziyi616**
Note: this requires MMCV>=1.3.1!

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The required versions of MMCV and MMDetection for different versions of MMDetect
 
 | MMDetection3D version | MMDetection version |    MMCV version     |
 |:-------------------:|:-------------------:|:-------------------:|
-| master              | mmdet>=2.10.0       | mmcv-full>=1.2.4, <=1.4|
+| master              | mmdet>=2.10.0       | mmcv-full>=1.3.1, <=1.4|
```
Comment (Tai-Wang) [inline docs/getting_started.md:15]: Please also modify this [file](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/__init__.py) correspondingly.

Comment (Wuziyi616) [reply docs/getting_started.md:15]: Done.

Comment (ZwwWayne) [reply docs/getting_started.md:15]: 1.3.2

File: mmdet3d/__init__.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ def digit_version(version_str):
     return digit_version
 
 
-mmcv_minimum_version = '1.2.4'
+mmcv_minimum_version = '1.3.1'
```
Comment (ZwwWayne) [inline mmdet3d/__init__.py:19]: 1.3.2

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**